shared/fs/common/path-item-action/header.js

```diff
@@ -33,36 +32,48 @@ class Header extends React.PureComponent<Props> {
   }
   render() {
     return (
-      <Kb.Box2 direction="vertical" fullWidth={true} centerChildren={true} style={styles.container}>
-        <PathItemIcon path={this.props.path} size={48} style={styles.pathItemIcon} />
-        {Styles.isMobile && <StaticBreadcrumb pathElements={Types.getPathElements(this.props.path)} />}
-        <Kb.Box2 direction="horizontal" style={styles.nameTextBox}>
-          <CommaSeparatedName
-            type="BodySmallSemibold"
-            name={Types.getPathName(this.props.path)}
-            elementStyle={Styles.collapseStyles([
-              styles.stylesNameText,
-              {color: Constants.getPathTextColor(this.props.path)},
-            ])}
-          />
+      <Kb.Box
```

Comment:
Why a `<Box>`? I was told to only use `Box2` in new code - is that the case?

Reply:
I think the general rule of thumb is if you need a box that's a flex column or flex row, or need to otherwise implement other Box2 features yourself, use Box2. Otherwise, can still use Box.

In this case, we need this dummy `onClick` prop which Box2 doesn't have yet, so it'd have to be a Box. If Box2 had `onClick`, it could have been just on the Box2 below.